### PR TITLE
chore(readme): update README: no longer beta

### PR DESCRIPTION
**Pull request by alexcastillo**
I don't think I should have the honor of submitting this PR but given how much work you all have on your plates, here's a bit of help with a very small, yet very important chore.

Congratulations on behalf of the community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Angular
 Angular is a development platform for building mobile and desktop web applications. This is the
 repository for [Angular 2][ng2], both the JavaScript (JS) and [Dart][dart] versions.
 
-Angular 2 is currently in **Beta**.
+Angular 2 is currently in **Release Candidates**.
```
Comment (vitor-faiante) [inline README.md]: @alexcastillo I believe it would be more accurate to say "Release Candidate" instead of "Release Candidates"

Comment (alexcastillo) [inline README.md]: @vitor-faiante, just using their own words: https://twitter.com/angularjs/status/727348250798227456

Comment (vitor-faiante) [inline README.md]: @alexcastillo Ahh I see, interesting that they used "Release Candidates" in the tweet and in the changelog says "Release Candidate". Now I'm confused

Comment (fxck) [inline README.md]: imo it's a typo in that tweet

Comment (vitor-faiante) [inline README.md]: I agree with you

Comment (alexcastillo) [inline README.md]: Fixed.

**Pull request merged**